pkg/plugins/runtime/k8s/webhooks/injector/injector_test.go

```diff
@@ -630,21 +630,5 @@ spec:
                   kuma.io/sidecar-injection: enabled`,
 			cfgFile: "inject.config.yaml",
 		}),
-		Entry("31. Pod non-existing patch in annotation", testCase{
-			num: "31",
-			mesh: `
-              apiVersion: kuma.io/v1alpha1
-              kind: Mesh
-              metadata:
-                name: default`,
-			namespace: `
-              apiVersion: v1
-              kind: Namespace
-              metadata:
-                name: default
-                annotations:
-                  kuma.io/sidecar-injection: enabled`,
-			cfgFile: "inject.config.yaml",
-		}),
```

Comment:
Should this be replaced by a test to check there's an error?

Reply:
I'm exactly working on this as we speak :-D

Reply:
done